rest/handler.go

```diff
@@ -189,6 +189,10 @@ func (h *handler) invoke(method handlerMethod) error {
 
 	h.logRequestLine()
 
+	if base.LogKeys["HTTP++"] {
```

Comment:
I think you're already checking the log key in logRequestBody (and doing a safe/synchronized map read there).